build.gradle

```diff
@@ -70,8 +70,8 @@ subprojects {
         // neo4jDockerImage system property is used in TestContainerUtil
         systemProperties 'user.language' : 'en' ,
                 'user.country ' : 'US',
-                'neo4jDockerImage' : project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-enterprise-debian' : 'neo4j:4.4.20-enterprise',
-                'neo4jCommunityDockerImage': project.hasProperty("neo4jDockerVersionOverride") ? 'neo4j:' + project.getProperty("neo4jDockerVersionOverride") + '-debian': 'neo4j:4.4.20'
```

Comment:
I don't think you should change these. neo4j 4.4.22 is available in maven, doesn't mean it will in docker hub

Reply:
Oh I was just following what the release card said and it had linked this: https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3471/commits/fd170d4c31ca8dfbfe5d46a2f4bb20f5d6d84288